packages/ui/src/documents/Nfts.tsx

```diff
@@ -114,6 +114,12 @@ export const Nfts: FunctionComponent<Props> = (props) => {
     return actions
   };
 
+  const renderNodeUrl = () => {
+    return window['__ETH_NETWORK__'] === 'kovan' ?
+        <a href={"https://kovan.etherscan.io/address/0x44a0579754D6c94e7bB2c26bFA7394311Cc50Ccb"}>here.</a> :
```

Comment:
How about adding a new env variable / window constant for the __NODE_ADDRESS__(name can be better)
```
getEthScanNodeUrl = () => {
  return `https://${window.__ETH_NETWORK__=== mainnet ? '': window.__ETH_NETWORK__ }etherscan.io/address/window.___NODE_ADDRESS__`
}
```
Also I will not return the `<a>` but just the url as we can use that in a context where it is not in a link. Maybe it can be a on click trigger

Reply:
Maybe we can also use some of this functions 
https://github.com/centrifuge/gateway/blob/5950fc6b239f5f4dba75a341980bfc1860817588/packages/lib/src/utils/etherscan.ts#L23

Reply:
i would not set the env variable for now bc this is not intended to stay as is, only until we get the tx hashes from the node job id (this is planned to be specced out next week). will update to use getAddressLink 👍